test/project/10002

```diff
@@ -1,10 +1,14 @@
-[
-  {
-    "code": "789c2d8c410e82301444e7f7574860039cc0780e371ec113d0aa5d34da8a95623446af6e8b2c665ef23219e8bb2300d44a05fa28face050588b560e3a7ea9126bdf5431c415994c1dca20d06822503bce274506e651b8c3ef5433c5cec71d98beeef829df46816c9c5ebecb4f5908533ee1a9e02d5860beca8419d80846e46ce3beb7d43f50f7f402616",
-    "vmType": "zkwasm",
-    "output": {
-      "type": "stdout"
-    },
-    "version": "0.1"
-  }
-]
\ No newline at end of file
+{
+  "datasourceURI": "postgres://test_user:test_passwd@localhost:5432/test?sslmode=disable",
```

Comment:
"postgres://risc0server-ro:[Nxr7pHMhwKBiWCxC@34.28.106.125:5432/risc0server?sslmode=disable](mailto:Nxr7pHMhwKBiWCxC@34.28.106.125:5432/risc0server?sslmode=disable)"
use this datasource URL,it's testnet uri

Reply:
those file will not used in the testnet, just local environment